### PR TITLE
Add hello2

### PR DESCRIPTION
**Pull request by tnarayanan**

### CODE REVIEW

**Review by amoghs1912 — APPROVED** [review]

**Review by pritishbudhrani — APPROVED** [review]
Looks good

**Review by Angelaq621 — APPROVED** [review]
Looks good

**Review by junsangYoon — APPROVED** [review]
Looks good

File: hello2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello2
```
Comment (Angelaq621) [inline hello2.txt:1]: Missing semicolon

**Pull request merged**